NITK presentations (#138)
subnero1/subnero1.github.io: _posts/2020-12-17-Subnero-talks-at-the-Open-source-Network-Experimentation.md

---
layout: post
title: Subnero talks at the Open-source Network Experimentation
banner : images/banner-pulse-standard.jpg
date:  2020-12-17
categories: wnc
thumbnail: images/pulse-smartsea.jpg
---

> For the second year consecutive, the Subnero team attended the virtual workshop organised by the [Department of Computer Science & Engineering of the National Institute of Technology Karnataka](https://cse.nitk.ac.in/.

Reviewer: ```suggestion
For the second consecutive year, the Subnero team attended the virtual workshop organised by the [Department of Computer Science & Engineering of the National Institute of Technology Karnataka](https://cse.nitk.ac.in/){:target="_blank"}.
```

> The objectives of the workshop were to give participants an understanding of the basics of underwater communications, to build underwater communication topologies and setting up simulationsin UnetStack amongst others.

Reviewer: ```suggestion
The objectives of the workshop were to give participants an understanding of the basics of underwater communications, to build underwater communication topologies and setting up simulations in UnetStack amongst others.
```

> To that effect, on Wednesday 16th of December, Dr Shiraz Shahabudeen did an overview of underwater communications, whilst Ms Florie Mancel talked about Subnero modems and solutions.

Reviewer: ```suggestion
To that effect, on Wednesday 16th of December, [Dr Shiraz Shahabudeen](https://subnero.com/people/shiraz/) did an overview of underwater communications, whilst Ms Florie Mancel talked about Subnero modems and solutions.
```

> On Thursday 17th of December, Dr Prasad Anjangi did a presentation on underwater protocols layers.

Reviewer: ```suggestion
On Thursday 17th of December, [Dr Prasad Anjangi](https://subnero.com/people/prasad/) did a presentation on underwater protocols layers.
```
